flaps/flaps.go

```diff
@@ -31,13 +33,36 @@ const headerFlyRequestId = "fly-request-id"
 
 type Client struct {
 	app        *api.AppCompact
-	peerIP     string
+	baseUrl    *url.URL
 	authToken  string
 	httpClient *http.Client
 	userAgent  string
 }
 
 func New(ctx context.Context, app *api.AppCompact) (*Client, error) {
+	cfg := config.FromContext(ctx)
+	if strings.TrimSpace(strings.ToLower(cfg.FlapsBaseURL)) == "peer" {
+		return newWithUsermodeWireguard(ctx, app)
+	}
+	flapsUrl, err := url.Parse(cfg.FlapsBaseURL)
+	if err != nil {
+		return nil, fmt.Errorf("invalid FLAPS_BASE_URL '%s' with error: %w", cfg.FlapsBaseURL, err)
```

Comment:
I think this should be 
```suggestion
		return nil, fmt.Errorf("invalid FLY_FLAPS_BASE_URL '%s' with error: %w", cfg.FlapsBaseURL, err)
```